packages/rest/src/validation/default-options.ts

```diff
@@ -0,0 +1,7 @@
+import {RequestBodyValidationOptions} from '../types';
+
+export const defaultValidationOptions: RequestBodyValidationOptions = {
+  $data: true,
+  ajvKeywords: true,
+  ajvErrors: true,
+};
```

Comment:
Is this fully backwards compatible? When existing projects update to the new version of `@loopback/rest`, will the validation behave the same way (accept the same set of input values, reject the same set of input values) and report the same validation errors as before?

Reply:
Not fully compatible, but the incompatibility will be only at the "message text" level, AFAIK.

If someone had this code:

```ts
@property({
  type: 'string',
  required: true,
  jsonSchema: {
    maxLength: 10,
    minLength: 1,
    errorMessage: 'Name should be between 10 and 30 characters.',
  },
})
```
They would have never gotten "'Name should be between 10 and 30 characters.'", instead they would see the default AJV validation error message.

With the changes in the PR, they will now see "Name should be between 10 and 30 characters.'".

Technically it is a breaking change, but the impact should be minimal for average scenarios.